packages/ckeditor5-dev-env/lib/release-tools/utils/versions.js

```diff
@@ -17,11 +17,19 @@ const versions = {
 	 */
 	getLastFromChangelog() {
 		const changelog = changelogUtils.getChangelog();
-		const regexp = /\n## \[?([^\]]+)/;
 
+		if ( !changelog ) {
+			return null;
+		}
+
+		const regexp = /\n## (\[([^\]]+)|[^ ]+)/;
 		const matches = changelog.match( regexp );
 
-		return matches ? matches[ 1 ] : null;
+		if ( !matches ) {
+			return null;
+		}
+
+		return matches[ 2 ] ? matches[ 2 ] : matches[ 1 ];
```

Comment:
This condition should be unnecessary. It wasn't necessary when I patched this in https://github.com/ckeditor/ckeditor5-dev/compare/t/148?expand=1.